avionics/common/include/states/main_descent.h

```diff
@@ -2,10 +2,14 @@
 
 /*Includes------------------------------------------------------------*/
 #include "calculations.h"
+#include "cameras.h"
 #include "state_input_struct.h"
 #include "state_interface.h"
+#include "tasks/restart_camera.hpp"
+#include "scheduler.hpp"
```

Comment:
It's possible to preregister a task (provide the scheduler with task details but not have it actually schedule the task until later), then manually schedule it later - I'd recommend skimming through the actual RocketScheduler header file; I've tried to keep good documentation for the public API.